shop4All-backend/src/main/java/com/example/shop4All_backend/entities/Image.java

```diff
@@ -0,0 +1,27 @@
+package com.example.shop4All_backend.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "image_model")
+@Getter
+@Setter
+public class Image {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long imageId;
+    private String name;
+    private String type;
+    @Column(length = 5000000)
+    private byte[] imageByte;
+    
+    public Image() {}
```

Comment:
You can get rid of this line of code (line 20) by adding @NoArgsConstructor annotation above the class.